.gitignore

```
@@ -0,0 +1,20 @@
+# Backup files and other machine-local stuff
+*.user
+*.svn
+*.autosave
+*.dropbox.attr
+*~
+*.orig
+*.swp
+*.old
+\#*\#
+# All dot files, if you wish to track one, just add it
+.*
+
+# Misc
+*Thumbs.db
+
+# Output Directories
+Bundle*/
+build*/
+cmake-build-*/
```

missing new line at the end of file ...
```suggestion
cmake-build-*/

```